catalog/app/containers/Bucket/PackageDialog/PackageDialog.js

```diff
@@ -268,6 +316,51 @@ export function MetaInput({ schemaError, input, meta, schema }) {
     changeText,
   ])
 
+  const { push: notify } = Notifications.use()
+  const [locked, setLocked] = React.useState(false)
+  // used to force json editor re-initialization
+  const [jsonEditorKey, setJsonEditorKey] = React.useState(1)
+
+  const onDrop = React.useCallback(
+    ([file]) => {
+      if (file.size > MAX_META_FILE_SIZE) {
+        notify(
+          <>
+            File too large ({readableBytes(file.size)}), must be under{' '}
+            {readableBytes(MAX_META_FILE_SIZE)}.
+          </>,
+        )
+        return
+      }
+      setLocked(true)
+      readFile(file)
+        .then((contents) => {
+          try {
+            JSON.parse(contents)
+          } catch (e) {
+            notify('The file does not contain valid JSON')
```

Comment:
Notification is almost invisible. I found it because I seek for it, after reading code.

Is it a temporary solution and do you want to show error inside Form?

Reply:
> Notification is almost invisible. I found it because I seek for it, after reading code.

well, it's a default MUI snackbar-style notification, we use those across the app to tell the user something has happened

> Is it a temporary solution and do you want to show error inside Form?

it _is_ shown as a regular field error (tho only after submit, like the rest of the field errors)